data/mods/TEST_DATA/lang/.gitignore

```diff
@@ -0,0 +1 @@
+mo/
```

Comment:
Maybe exclude `INVALID_RAND.mo` from .gitignore?

Reply:
What does git do when someone else modifies the added but ignored .mo files? Do they show up in the modified files list? If not people might forget to commit them when they are changed.

Reply:
It does get listed:
```
[brett] ~/Cataclysm-DDA [mo_loader]  % vim data/mods/TEST_DATA/lang/mo/ru/LC_MESSAGES/INVALID_RAND.mo 
[brett] ~/Cataclysm-DDA [mo_loader] M % git status
On branch mo_loader
Your branch is up to date with 'brett/mo_loader'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   data/mods/TEST_DATA/lang/mo/ru/LC_MESSAGES/INVALID_RAND.mo

no changes added to commit (use "git add" and/or "git commit -a")
```